[kubernetes resources] add cluster name setting instructions (#8346)
DataDog/documentation: content/en/infrastructure/livecontainers.md

To enable Kubernetes Resources for Live Containers, follow the [Helm instructions][10] and add the following changes to your `values.yaml` file:

{{< code-block lang="yaml" >}}
datadog:
  ...
  processAgent:
    enabled: true
  ...
  orchestratorExplorer:
    enabled: true
...
clusterAgent:
  enabled: true
  image:
    repository: datadog/cluster-agent
    tag: latest
    pullPolicy: Always
...
agents:
  image:
    repository: datadog/agent
    tag: latest
    pullPolicy: Always
...
{{< /code-block >}}

In cases where the agent is not able to automatically detect the Kubernetes cluster name, you must set it in `values.yaml` as well:

{{< code-block lang="yaml" >}}
datadog:
   ...
   clusterName: "<your Kubernetes cluster's name>"
   ...
{{< /code-block >}}

> On Google's GKE, AWS EKS, and Azure AKS, this is unnecessary, unless the agent and the cluster agent don't have access to the cloud metadata APIs.

Author: we'll relax this constraint, but yeah i'll add it for now

To enable Kubernetes Resources for Live Containers, follow the [Helm instructions][10] and add the following changes to your `values.yaml` file:

> {{< code-block lang="yaml" >}}

Reviewer: ```suggestion
{{< code-block lang="yaml" filename="values.yaml" >}}
```

datadog:
  ...
  processAgent:
    enabled: true
  ...
  orchestratorExplorer:
    enabled: true
...
clusterAgent:
  enabled: true
  image:
    repository: datadog/cluster-agent
    tag: latest
    pullPolicy: Always
...
agents:
  image:
    repository: datadog/agent
    tag: latest
    pullPolicy: Always
...
{{< /code-block >}}

> In cases where the agent is not able to automatically detect the Kubernetes cluster name, you must set it in `values.yaml` as well:

Reviewer: ```suggestion
In cases where the Agent is not able to automatically detect the Kubernetes cluster name, you must set it in `values.yaml`:
```

> {{< code-block lang="yaml" >}}

Reviewer: ```suggestion
{{< code-block lang="yaml" filename="values.yaml" >}}
```

datadog:
   ...
   clusterName: <PLACEHOLDER>
   ...
{{< /code-block >}}

> **Note**: the cluster name must be 40-character long or less. We are working on relaxing this constraint.

Reviewer: ```suggestion
**Note**: The cluster name must be 40-characters or less.
```

> On Google's GKE, AWS EKS, and Azure AKS, this is unnecessary, unless the agent and the cluster agent don't have access to the cloud metadata APIs, or the cluster name is longer than 40 characters.

Reviewer: ```suggestion
On Google's GKE, AWS EKS, and Azure AKS, this is unnecessary, unless the Agent and the cluster Agent don't have access to the cloud metadata APIs, or the cluster name is longer than 40 characters.
```
